pkgs/tools/misc/plantuml-server/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "plantuml-server";
+  version = "1.2020.15";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/plantuml/${version}/plantuml.${version}.war";
```

Comment:
* It would be better to build .war from source
* There is https://github.com/plantuml/plantuml-server

Reply:
done